### PR TITLE
Manual.md: relax and clarify the "no sripts" policy

### PR DESCRIPTION
**Pull request by ahesford**
I think our "no scripts" policy is too strict and precludes the inclusion of useful administrative tools. These are helpful to have packaged because it makes deploying new Void systems easier by keeping tools in one place. Based on a discussion in #xbps, I propose the following relaxed language that still gives us a reasonable justification for rejecting apparently useless scripts while admitting some more useful ones.

### CODE REVIEW

File: Manual.md [hunk 1]
```diff
@@ -136,10 +136,11 @@ the Void packages system.
 1. Required: Another package either within the repository or pending inclusion
    requires the package.
 
-In particular, new themes and small shell scripts are highly unlikely
-to be accepted. New fonts are welcome if they provide value beyond
-aesthetics (e.g. they contain glyphs for a script missing in already
-packaged fonts).
+In particular, new themes are highly unlikely to be accepted. Simple shell
+scripts are unlikly to be accepted unless they provide considerable value to a
```
Comment (sgn) [inline Manual.md:140]: s/unlikly/unlikely

**Review by Chocimier — APPROVED** [review]
typo in "no sripts" in commit message

File: Manual.md [hunk 2]
```diff
@@ -136,10 +136,10 @@ the Void packages system.
 1. Required: Another package either within the repository or pending inclusion
    requires the package.
 
-In particular, new themes and small shell scripts are highly unlikely
-to be accepted. New fonts are welcome if they provide value beyond
-aesthetics (e.g. they contain glyphs for a script missing in already
-packaged fonts).
+In particular, new themes are highly unlikely to be accepted. Simple shell
+scripts are unlikely to be accepted unless they provide considerable value to a
+broad user base. New fonts are welcome if they provide value beyond aesthetics
```
Comment (the-maldridge) [inline Manual.md:141]: New fonts are not welcome, but might be accepted.

**Review by the-maldridge — CHANGES_REQUESTED** [review]

**Comment by ahesford** [comment]
I've seen no objections except for rewording suggestions that have been accommodated, so merging seems appropriate.

**Pull request closed without merging**